datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/GmsGraphQLEngine.java

```diff
@@ -74,15 +78,16 @@
     public static final MLModelType ML_MODEL_TYPE = new MLModelType(GmsClientFactory.getMLModelsClient());
     public static final DataFlowType DATA_FLOW_TYPE = new DataFlowType(GmsClientFactory.getDataFlowsClient());
     public static final DataJobType DATA_JOB_TYPE = new DataJobType(GmsClientFactory.getDataJobsClient());
+    public static final GenericDownstreamLineageType GENERIC_DOWNSTREAM_LINEAGE_TYPE =
+            new GenericDownstreamLineageType(GmsClientFactory.getGenericDownstreamLineagesClient());
 
     /**
      * Configures the graph objects that can be fetched primary key.
      */
-    public static final List<LoadableType<?>> LOADABLE_TYPES = ImmutableList.of(
+    public static final List<EntityType<?>> ENTITY_TYPES = ImmutableList.of(
```

Comment:
Do we consider GenericDownstreamLineageType to be an EntityType?

Reply:
I do not- an instance of GenericDownstreamLineage does not have an urn